ddl/multi_schema_change.go

```diff
@@ -254,6 +254,7 @@ func fillMultiSchemaInfo(info *model.MultiSchemaInfo, job *model.Job) (err error
 	case model.ActionSetDefaultValue:
 		col := job.Args[0].(*table.Column)
 		info.ModifyColumns = append(info.ModifyColumns, col.Name)
+	case model.ActionAlterIndexVisibility:
```

Comment:
It should be added into a slice to avoid `drop index i` with `invisible index i`. But we didn't have such slices now.